packages/widgetbook_annotation/README.md

```diff
@@ -8,291 +8,4 @@
 
```

Comment:
Widgetbook is a flutter package which helps developers cataloguing their widgets, testing them quickly on multiple devices and themes, and sharing them easily with designers and clients. Inspired by Storybook.js.